src/nxt_process.c

```diff
@@ -6,6 +6,10 @@
 
 #include <nxt_main.h>
 
+#if (NXT_HAVE_CGROUP)
```

Comment:
I would include our own headers unconditionally.

Reply:
Done.

Reply:
Also in src/nxt_process.h

Reply:
Actually the one in src/nxt_process.h is required as we only want to call nxt_isolation_set_cgroup() when cgroups are supported, just like all the function calls in  nxt_isolation_set() which are called conditionally.